src/GraphQL/Types/Composite/AutoRegisteringOutputHelper.cs

```diff
@@ -26,14 +29,18 @@ internal static class AutoRegisteringOutputHelper
         FieldType fieldType,
         Func<MemberInfo, LambdaExpression> BuildMemberInstanceExpression,
         Func<Type, Func<FieldType, ParameterInfo, ArgumentInformation>> getTypedArgumentInfoMethod,
-        Action<ParameterInfo, QueryArgument> ApplyArgumentAttributes)
+        Action<ParameterInfo, QueryArgument> ApplyArgumentAttributes,
+        bool isInterface)
```

Comment:
This change (to an internal class) skips building field resolver for interface fields, and instead assigns a static field resolver implementation that throws an exception.

Skipping building the field resolvers saves a little time during generation of the schema, as compilation is relatively slow.